### ISSUE TITLE
Install error 

### ISSUE THREAD

**Issue by LliliansCalvo** [Bioconductor/DelayedArray#103]
I am trying to install GenomicFeatures but I received and error that seems to be related with the version of DelayedArray. I need version 0.26 but I cant load it on my mac. 

```
BiocManager::install("GenomicFeatures", force= TRUE)
library("GenomicFeatures")
Error: package or namespace load failed for ‘GenomicFeatures’ in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 namespace ‘DelayedArray’ 0.25.0 is being loaded, but >= 0.26.1 is required

sessionInfo() 
R version 4.3.0 (2023-04-21)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Ventura 13.3.1

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.11.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

other attached packages:
 [1] DelayedArray_0.25.0   MatrixGenerics_1.12.0 matrixStats_0.63.0   
 [4] Matrix_1.5-4          BiocManager_1.30.20   AnnotationDbi_1.62.1 
 [7] Biobase_2.60.0        GenomicRanges_1.52.0  GenomeInfoDb_1.36.0  
[10] IRanges_2.34.0        S4Vectors_0.38.1      BiocGenerics_0.46.0  
```

I can see I have DelayedArray_0.25.0 but I need 0.26, however when I try to install it it doesn't work
```
BiocManager::install("DelayedArray")

Warning: unable to access index for repository https://bioconductor.org/packages/3.17/data/annotation/bin/macosx/big-sur-arm64/contrib/4.3:
  cannot open URL 'https://bioconductor.org/packages/3.17/data/annotation/bin/macosx/big-sur-arm64/contrib/4.3/PACKAGES'
trying URL 'https://bioconductor.org/packages/3.17/bioc/bin/macosx/big-sur-arm64/contrib/4.3/DelayedArray_0.25.0.tgz'
```

```
BiocManager::install("DelayedArray_0.26.1", version = "3.17")
'getOption("repos")' replaces Bioconductor standard repositories, see
'help("repositories", package = "BiocManager")' for details.
Replacement repositories:
    CRAN: https://cran.rstudio.com/
Bioconductor version 3.17 (BiocManager 1.30.20), R 4.3.0 (2023-04-21)
Installing package(s) 'DelayedArray_0.26.1'
Warning: unable to access index for repository https://bioconductor.org/packages/3.17/data/annotation/bin/macosx/big-sur-arm64/contrib/4.3:
  cannot open URL 'https://bioconductor.org/packages/3.17/data/annotation/bin/macosx/big-sur-arm64/contrib/4.3/PACKAGES'
Old packages: 'AnnotationFilter', 'AnnotationHub', 'BiocParallel',
  'DelayedArray', 'ensembldb', 'GenomicFeatures', 'httpuv', 'httr', 'httr2',
  'interactiveDisplayBase', 'ProtGenerics', 'spatstat.geom', 'spatstat.utils',
  'waldo', 'wk'
Update all/some/none? [a/s/n]: 
a
Warning: unable to access index for repository https://bioconductor.org/packages/3.17/data/annotation/bin/macosx/big-sur-arm64/contrib/4.3:
  cannot open URL 'https://bioconductor.org/packages/3.17/data/annotation/bin/macosx/big-sur-arm64/contrib/4.3/PACKAGES'

...


trying URL 'https://bioconductor.org/packages/3.17/bioc/bin/macosx/big-sur-arm64/contrib/4.3/DelayedArray_0.25.0.tgz'
Content type 'application/x-gzip' length 2652021 bytes (2.5 MB)
```

**Comment by lshep** [comment]
Please don't use `force=TRUE` on a regular basis. That should only be used in very rare circumstances.  
The later version of DelayedArray has yet to build and propagate on our build system.  As indicated by the version on the [build report for mac arm64](https://bioconductor.org/checkResults/3.17/bioc-mac-arm64-LATEST/) as well as the [DelayedArray landing page](https://bioconductor.org/checkResults/3.17/bioc-mac-arm64-LATEST/).  It looks like the build report page has not been updated since April 28th  @jwokaty can you comment on the status of the builds please?

**Comment by hpages** [comment]
Looks like **DelayedArray** 0.26 is finally available on all platforms as of today in BioC 3.17: 
https://bioconductor.org/packages/3.17/DelayedArray

@LliliansCalvo Can you try to install again with `BiocManager::install("DelayedArray")`?
